<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TareaForm</class>
 
 <!-- Ventana con centralizado -->
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="main_layout">


    <!-- Formulario -->
    <!-- Id -->
    <item>
    <layout class="QFormLayout" name="formLayout">
        
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>Id</string>
       </property>
       
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>

      </widget>
     </item>
     
     <item row="0" column="1">
      <widget class="QLineEdit" name="entry_id">

       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>

      </widget>
     </item>
    

    <!-- Descripcción -->
     <item row="1" column="0">
      <widget class="QLabel" name="label_description">
       <property name="text">
        <string>Descripcción</string>
       </property>
      </widget>
     </item>
     
     <item row="1" column="1">
      <widget class="QLineEdit" name="entry_description">
      </widget>
     </item>
    

    <!-- Baja Checkbox -->
     <item row="2" column="0">
      <widget class="QLabel" name="label_soft_delete">
       <property name="text">
        <string>Baja</string>
       </property>
      </widget>
     </item>
     
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkbox_soft_delete">
      </widget>
     </item>

    </layout>
    </item>
    
    
    <!-- Agragar y actualizar -->
    <item>
    <layout class="QHBoxLayout">
     
     <item>
      <widget class="QPushButton" name="button_save">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text"><string>Cancel</string></property>
      </widget>
     </item>

    </layout>
    </item>
    

    <!-- Tabla -->
    <item>
     <widget class="QTableWidget" name="table">
     </widget>
    </item>
    
   </layout>
  </widget> <!-- Central widget para centrar todo -->

 <resources/>
 <connections/>
</ui>